File: engine/engine-model/src/main/scala/tools/packet/transfer/name/NameChangeError.scala
package tools.packet.transfer.name

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class NameChangeError(private var _error: Int) extends PacketInput {
  def error: Int = _error

  override def opcode(): SendOpcode = SendOpcode.CASH_SHOP_CHECK_NAME_CHANGE_POSSIBLE_RESULT
}

File: engine/engine-model/src/main/java/tools/packet/transfer/name/NameChangeError.java
package tools.packet.transfer.name;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record NameChangeError(Integer error) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.CASH_SHOP_CHECK_NAME_CHANGE_POSSIBLE_RESULT;
   }
}